tests/Unit/Controllers/Store/BundleControllerTest.php

```diff
@@ -397,4 +397,32 @@ public function testICannotDisburseAnEmptyBundle()
             ->seePageIs($route)
             ->assertResponseStatus(200);
     }
+
+    /** @test */
+    public function itCanAcceptAndCleanVouchersWithSpacesIn()
```

Comment:
Do you also want to test that `null` and `""` values will get cleared from the list?